internal/hud/webview/convert.go

```diff
@@ -143,6 +121,11 @@ func tiltfileResourceView(s store.EngineState) Resource {
 }
 
 func resourceInfoView(mt *store.ManifestTarget) ResourceInfoView {
+	if mt.Manifest.Name == model.UnresourcedYAMLManifestName {
```

Comment:
i wonder if the check here shouldn't be for the name, but rather for k8s manifests that have no image targets

Reply:
i don't think so. we might have k8s manifest w/o image target that ARE still workloads, i.e. we want to display them like any other manifest; we only want this special view for the non-pod-creating leftover yaml.